app/mailers/application_mailer.rb

```diff
@@ -4,6 +4,31 @@ class ApplicationMailer < ActionMailer::Base
 
   private
 
+  def send_sendgrid_email(dynamic_template_data:, template_id:, to:)
+    sendgrid_api
+      .client
+      .mail
+      ._("send")
+      .post(request_body: sendgrid_request_body(dynamic_template_data:, template_id:, to:))
+  end
+
+  def sendgrid_api
+    @sendgrid_api ||= SendGrid::API.new(api_key: ENV["SENDGRID_API_KEY"])
+  end
+
+  def sendgrid_request_body(dynamic_template_data:, template_id:, to:)
+    {
+      dynamic_template_data:,
+      from: {
+        email: "no-reply@talentprotocol.com",
+        name: "Filipe at Talent Protocol"
```

Comment:
I've created this env var so we can distinguish emails from prod and staging. I've added in both environments. Can you add it o the `.env` file?

```suggestion
        name: ENV['EMAILS_FROM']
```

Reply:
Okay sounds good!